.env

```
@@ -9,7 +9,8 @@ _APP_CONSOLE_WHITELIST_IPS=
 _APP_CONSOLE_COUNTRIES_DENYLIST=AQ
 _APP_CONSOLE_HOSTNAMES=localhost,appwrite.io,*.appwrite.io
 _APP_SYSTEM_EMAIL_NAME=Appwrite
-_APP_SYSTEM_EMAIL_ADDRESS=team@appwrite.io
+_APP_SYSTEM_EMAIL_ADDRESS=noreply@appwrite.io
+_APP_SYSTEM_TEAM_EMAIL=teams@appwrite.io
```

```suggestion
_APP_SYSTEM_TEAM_EMAIL=team@appwrite.io
```
it should be team right?